src/components/templates/ClaimsList/index.tsx

```diff
@@ -25,132 +25,186 @@ export default function ClaimsList(props){
+        actionButton: (
+          <Button
+            attributes={{ className: 'govuk-!-margin-top-4 govuk-!-margin-bottom-4' }}
+            variant='secondary'
+            onClick={() => {
+              _rowClick(item);
+            }}
+          >
+            {buttonContent}
+          </Button>
+        ),
+        status: statusMapping(item.pyStatusWork)
       };
 
-      if(item.Claim.ChildrenJSON){
+      if (item.Claim.ChildrenJSON) {
         const additionalChildren = extractChildren(item.Claim.ChildrenJSON);
-        additionalChildren.forEach(child =>{
+        additionalChildren.forEach(child => {
           const newChild = {
-            firstName : child.name,
-            lastName : ' ',
-            dob : child.dob ? GBdate(child.dob) : ''
-          }
+            firstName: child.name,
+            lastName: ' ',
+            dob: child.dob ? GBdate(child.dob) : ''
+          };
           claimItem.children.push(newChild);
-        })
-      }else{
+        });
+      } else {
         claimItem.children.push({
-          firstName : item.Claim.Child.pyFirstName,
-          lastName : item.Claim.Child.pyLastName,
-          dob : item.Claim.Child.DateOfBirth ? GBdate(item.Claim.Child.DateOfBirth) : ''
+          firstName: item.Claim.Child.pyFirstName,
+          lastName: item.Claim.Child.pyLastName,
+          dob: item.Claim.Child.DateOfBirth ? GBdate(item.Claim.Child.DateOfBirth) : ''
         });
       }
       claimsData.push(claimItem);
-    })
+    });
+
     return claimsData;
   }
 
   useEffect(() => {
     setClaims([...getClaims()]);
-  },[data])
+  }, [data]);
 
-  return (
-    <>
-    
-      {claims.length !== 0 && <h2 className='govuk-heading-m'>{title}</h2>}
-    
-      {claims.length > 1 && <h3 className='govuk-heading-s'>{t('CHILDREN_ADDED')}</h3>}
-      {claims.map(claimItem =>
-    
-        <div className='govuk-summary-list inline-block-warning' key={claimItem.claimRef}>
-         {!caseId?.includes(claimItem.claimRef) &&( claimItem?.status?.text === 'In Progress' || claimItem?.status?.text === 'AR WAITH')  && <WarningText date ={claimItem?.dateUpdated}/>}
-          <div className='govuk-summary-list__row'>
-            <dt className='govuk-summary-list__key'>
-          
-              {claimItem.children.map(child =>
-              
-           
-                <p key={child.firstName}>
-                  {child.firstName && child.lastName && `${child.firstName} ${child.lastName}`}
-                  {child.dob && <><br/><span className='govuk-!-font-weight-regular'>{`${t('DATE_OF_BIRTH')} ${child.dob}`}</span><br/></>}
-                  <span className='govuk-!-font-weight-regular'>{t('CREATED_DATE')} {claimItem.dateCreated}</span>
-                </p>
-              )}
-              {claimItem.actionButton}
+  function renderChildDetails(claimItem) {
+    return claimItem.children.map((child, index) => (
+      <dl className='govuk-summary-list' key={Math.random()}>
```

Comment:
You should use index with child name instead of Math.random().

Reply:
I don't think I can use index as a key you get the react error - " error  Do not use Array index in keys  react/no-array-index-key"

I've changed it to be first name as the key which should be ok as I doubt you'd claim for children with the same first name.